gobblin-hive-registration/src/main/java/gobblin/hive/policy/HiveRegistrationPolicyBase.java

```diff
@@ -373,4 +380,36 @@ public static HiveRegistrationPolicy getPolicy(State props) {
           "Unable to instantiate " + HiveRegistrationPolicy.class.getSimpleName() + " with type " + policyType, e);
     }
   }
+
+  /**
+   *
+   * @param props Job-level state, props A {@link State} object that contains property,
+   * {@link #HIVE_REGISTRATION_POLICY}, which is the class name of the desired policy.
+   * This policy class must have a constructor that takes a {@link State} object.
+   * @param runtimeExtraState There are occasions where some runtime attributes need to be added into Hive Table's
```

Comment:
Please rephrase this doc, it is hard to understand. Is this parameter needed though? Can't both set of properties be combined in one.